Samples/Tests/Core/IntegrationTestBase.swift

```diff
@@ -109,12 +109,16 @@ class IntegrationTestBase: XCTestCase {
                         throw Error.tooManyScrolls
                     }
 
+                    #if os(watchOS)
+                    app.swipeUp(velocity: .init(floatLiteral: 100.0))
```

Comment:
Maybe there is an API for "Digital Crown" on Apple Watch?

Reply:
I've decided to remove UI interactions as those were flaky while not required